tests/integration_tests/functional/test_cpu_template_helper.py

```diff
@@ -350,44 +345,41 @@ def test_cpu_config_change(test_microvm_with_api, cpu_template_helper, tmp_path)
     # cpu_template_helper.fingerprint_dump(vm_config_path, baseline_path)
 
     # Compare with baseline
-    actual_cpu_config = json.loads(fingerprint_path.read_text(encoding="utf-8"))[
-        "guest_cpu_config"
-    ]
-    baseline_cpu_config = json.loads(baseline_path.read_text(encoding="utf-8"))[
-        "guest_cpu_config"
-    ]
-    assert actual_cpu_config == baseline_cpu_config
+    cpu_template_helper.fingerprint_compare(
+        baseline_path,
+        fingerprint_path,
+        filters,
+    )
 
 
-@pytest.mark.nonci
-def test_host_fingerprint_change(test_microvm_with_api, cpu_template_helper, tmp_path):
+@pytest.mark.skipif(
+    utils.get_kernel_version(level=1) not in SUPPORTED_HOST_KERNELS,
+    reason=f"Supported kernels are {SUPPORTED_HOST_KERNELS}",
+)
+def test_guest_cpu_config_change(test_microvm_with_api, tmp_path, cpu_template_helper):
     """
-    Verify that the host fingerprint has not changed since the baseline
+    Verify that the guest CPU config has not changed since the baseline
```

Comment:
Yes, there is. That is covered by `test_host_fingerprint_change`. We are having two separate tests `test_guest_cpu_config_change` and `test_host_fingerprint_change` for guest CPU config change and host environment change respectively, since the host environment change could happen asynchronously with our codebase and that should not block our CI.